### DISCUSSION TITLE
Welcome to BTCPay Server Discussions! Introduction thread.

### DISCUSSION THREAD

**Discussion by pavlenex** [btcpayserver/btcpayserver#2132] [General]
## 👋 Welcome to BTCPay Server Discussions!
Feel free to use this discussion thread to introduce yourself! 💚 

We're using discussions to have a better distinction between questions, feature request and actual issues (bug reports and to-do), so hopefully this helps us have a better structure.

  We’re using **Discussions** as a place to connect with other members of our community. We hope that you:
  * Ask questions
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

My name is Pavlenex, I have have been involved with the BTCPay Server project for 3 years now, just a month or two after @NicolasDorier started it and @rockstardev joined in. I've started as a casual contributor helping with support, documentation, videos, doing PR and marketing and slowly it got under my skin. I started contributing more and more time to the project and ended up dedicating more hours to BTCPay than to my regular job at that time. 
These days my focus is product managing the release cycle for the project. It's my passion to help developers and contributors work together and build with focus and to enjoy the process.

**Comment by rockstardev** [comment]
My name is RockstarDev, and I the Chief Philosopher Officer on the BTCPay Server project. I would say more, but it's 8:23 AM here in my timezone... so I'll reserve FIRST spot and edit this once I get few beers into the system.

**Reply by Zaxounette** [reply]
> so I'll reserve FIRST spot

That's dirty play !!!

**Reply by rockstardev** [reply]
Double dirty considering I did what I usually do - got few beers and forgot I was supposed to come back here and edit.

**Reply by Miladr7** [reply]
Gang

**Reply by Chrisselon** [reply]
i have no idea what the hell im doing i nee help i need to pay for what ever

**Reply by Homelow1** [reply]
Can some one help me pay for my bitcoin.

**Comment by Zaxounette** [comment]
Hi, I'm Zaxounette !

I've been part of the BTCPay family for more than a year now.

I currently write words for BTCPay Server, as opposed to code.
My activities relate to issue reports, expanding and updating the documentation, making visuals and video ressources, providing some low hanging support in the different channels.
I also moderate the BTCPay Server Directory.

**Comment by Eskyee** [comment]
Hi  I`m Esky
Positivity Engineer

I've also been part of the BTCPay family for more than three years now. since @NicolasDorier & @rockstardev started the BTCPay idea! working with this team has been a ride 💯 a proud btcpay contributor , while supporting btcpay mattermost community from time to time. a self-taught coder since this project began. i love digging out the open source code..  and learning new bitcoin knowledge everyday.

Offering BTCPay Jungle as a free hosted bitcoin payment platform to fast track noobs to the world of Bitcoin.. 2017 -2021
the focus is - JungleLab, BTCPay, Music. 24 7 365 - full time. 2021

It's my passion to help btcpay developers, artists, noobs & contributors work together and build with focus and learn to enjoy the process. I love ghosting on projects 🥇 my opinions are my Own.. reach out if needed. I always here. 

one love esky xx the future today





 
Run and use your OWN BTCPay full node. It’s not hard. 💚

**Reply by rockstardev** [reply]
Just everyone should keep in mind that Eskyee's positivity and love are so strong, that you must accept them. 

That and be ready to buy ticker and fly over for upcoming BtcPay music festival, ofc 🚀

**Comment by Bell2436** [comment]
I sent in $20 Bitcoin into my dashboard balance but I was credited $5 why

**Reply by Lancej777** [reply]
I don't understand any of this I can't even send bitcoin .don't know how or where mine are even at

**Reply by Miladr7** [reply]
> I sent in $20 Bitcoin into my dashboard balance but I was credited $5 why

For25% legit profit

**Comment by cassa72** [comment]
Umm

**Comment by cassa72** [comment]
Ok

**Comment by LilDaffy** [comment]
So far I've made 2 payments, but I still have to pay the commission fee for the second transfer. I am honestly kind of sceptical considering I've already made 2 payments and have not received anything. Can someone please tell me that I'm not wasting my time or money.

**Reply by pavlenex** [reply]
@LilDaffy If you're talking about mining bitcoin website, please do not pay, it's a SCAM.

**Reply by LilDaffy** [reply]
It's bonus-btc.online/offbitbonus
Like I said I've made 2 payments already that equals about 130 but it says the payment system  requests commission for the second transfer. Which is another 48 dollars.

**Reply by LilDaffy** [reply]
So I'm probably not gonna be able to get that 130 back huh. Welp I'm just gonna take that lesson right in the teeth. I should have known better.

**Reply by pavlenex** [reply]
Yes, unfortunately. If you could tell us the URL of the website, we can try to report them to the domain registrant so they can suspend them at least, though these scammers always just find alternative domain and keep scamming people. May I ask how did you find their website? Was it social media advertisement or something else?

**Reply by LilDaffy** [reply]
Multiple emails. And even had a file sent to my Google drive.

**Reply by LilDaffy** [reply]
https://safepay.website/i/aydCoSgcdz3S5ayfskhpW

**Comment by LilDaffy** [comment]
Is it possible to get a refund for Bitcoin payments made to a scam server that offered a bigger amount of Bitcoin for payment fees?

**Reply by pavlenex** [reply]
Sadly there is nothing we (BTCPay Server project) can do for you. The bitcoin bonus scam does use our software, but we don't have any control over their installation of it. BTCPay is a self-hosted software. You can report them to their domain registrant (we did this already) or simply local law enforcement.  https://docs.btcpayserver.org/FAQ/General/#what-if-i-have-a-problem-with-a-paid-invoice

**Comment by Alexander1690** [comment]
Hi Alex Hill here im looking for help in withdrawing my funds from BitCoin. I've been told to pay 64dollars but nothing let's me pay it ?Everything I've tried to get my funds after being told what to do it's the same result. Copied to clipboard?

**Reply by pavlenex** [reply]
@Alexander1690 It's a scam https://blog.btcpayserver.org/bitcoin-bonus-scam/#

**Comment by Chrisselon** [comment]
Welcome to my world 🌍 Sent from my iPhoneOn Dec 4, 2023, at 7:44 AM, Alexander1690 ***@***.***> wrote:﻿
Hi Alex Hill here im looking for help in withdrawing my funds from BitCoin. I've been told to pay 64dollars but nothing let's me pay it ?Everything I've tried to get my funds after being told what to do it's the same result. Copied to clipboard?

**Comment by Alexander1690** [comment]
Thanks for getting back to me I had a feeling it was a SCAM. THANKS FOR
CONFIRMING IT.ALEX.
